lib/httplib/httpheaders.go

```diff
@@ -99,6 +99,9 @@ func GetDefaultContentSecurityPolicy() string {
 		"frame-ancestors 'none'",
 		// additional default restrictions
 		"object-src 'none'",
+		// auto-pay plans in Cloud use stripe.com to manage billing information
+		"script-src 'self' https://js.stripe.com",
```

Comment:
Does this need to be applied globally to the `DefaultContentSecurityPolicy`?  Instead could we just append to this policy when a page specifically needs to allow stripe code?

Reply:
(With a few exceptions) the web UI is a single page app - the "pages" that use stripe are the same as any other part of the web UI, right?

Reply:
If Stripe is being included as part of the single page app I understand this may make sense.  But it sounded like this is new development, and if so, instead we should isolate the Stripe interaction on a different page.  We would like to avoid providing stripe this level of trust across our entire application.

Reply:
@michellescripts @mcbattirola since it's a golang server side file, can we do:

```golang
if plan == "team" {
   headers["script-src 'self'] = "https://js.stripe.com"
}
```

This change alone will massively reduce the blast radius from any OSS or Enterprise customer